android/src/main/java/it/innove/Peripheral.java

```diff
@@ -208,6 +213,10 @@ public boolean isConnected() {
         return connected;
     }
 
+    public boolean isDisconnected() {
+      return !isConnected() || gatt == null;
+    }
+
```

Comment:
Added a little utility method since we check these gatt and isConnected very often.